### DISCUSSION TITLE
Release `sea-query-binder 0.4` to crates.io?

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [SeaQL/sea-query#660] [General]
It appears like the update of `sea-query-binder 0.3 -> 0.4` is meant to go along the `sea-query 0.28 -> 0.29` update.

However, `sea-query-binder 0.4` is not yet released on crates.io.

Do I see this correctly? Do I need to wait for `sea-query-binder 0.4`  before updating to `sea-query 0.29`?

---

Side note: I would also like to update to `sqlx 0.7`. There is a PR (currently blocked): https://github.com/SeaQL/sea-query/pull/652
Will the merged PR be released with a future `sea-query 0.30`? Or rather `0.29.x`?

Thanks!

**Comment by tyt2y3** [comment]
As you may already know, we are in the middle of a release window.
0.29 is meant to be paired with binder 0.4, which will be released soon.
So the original plan was to make 0.29 compatible with both sqlx 0.6 and 0.7, but it was deemed not possible due to a dependency upgrade (`ipnetwork`, IIRC).
So what's likely to happen is after upgrading to sqlx 0.7, there will be a binder 0.5 and sea-query 0.30 release.
And hopefully we can got it all done next week.

**Reply by LeoniePhiline** [reply]
Thanks for your response! ❤️

I was unaware of the release window. 

Your response helped me understand that waiting for binder 0.4 and query 0.30 and then updating sqlx to 0.7 along with that will be the right way to go for my setup.

I had some trouble with query 0.29 combined with binder 0.3. The `bind_sqlx` methods were no longer defined and it looked like some create features were no longer forwarded to dependencies.

I'm not sure but I will stop trying to figure this out and continue with sea-query 0.30 once it's ready.

**Reply by LeoniePhiline** [reply]
sea-query-binder 0.4 works like a charm. 
My project was uncompilable on sea-query 0.29.1 with sea-query-binder 0.3 - so there must be an incompatibility.
After upgrading to the freshly released sea-query-binder 0.4, all checks and tests passed immediately.
Thank you! :)

**Comment by tyt2y3** [comment]
Please try https://github.com/SeaQL/sea-query/releases/tag/0.30.0
